extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -629,6 +785,9 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
 
   /** @override */
   letCreativeTriggerRenderStart() {
+    if (this.isInAdSizeOptimizationExperimentBranch()) {
```

Comment:
Can you add this in onCreativeRender?  It's a more explicit location of when creative rendering occurs as this function is more around analytics triggers and controlling if render should start.

Reply:
(done)